src/Files.App/Actions/Git/GitInitAction.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using Files.App.Contexts;
+
+namespace Files.App.Actions
+{
+    sealed class GitInitAction : ObservableObject, IAction
+    {
+		private readonly IContentPageContext _context;
+
+		public string Label
+			=> "Init".GetLocalizedResource();
+
+		public string Description
+			=> "InitRepositoryDescription".GetLocalizedResource();
+
+		public bool IsExecutable => 
+			(_context.SelectedItem is not null ||
```

Comment:
Is this one needed?

Reply:
Isn't initialization related to the previewed item? It might be the current folder as well as the selected one

Reply:
I think it should only be for the current directory

Reply:
@0x5bfa @hecksmosis @heftymouse fyi